src/app/(Authentication)/Login/Password/styles.ts

```diff
@@ -0,0 +1,76 @@
+import { StyleSheet } from 'react-native';
+
+export default StyleSheet.create({
+  container: {
+    backgroundColor: '#fff',
+    width: '100%',
+    height: '100%',
+    alignItems: 'center',
+    justifyContent: 'flex-start',
+  },
+  backText: {
+    color: '#000',
+    fontSize: 16,
+    opacity: 0.3,
+    fontWeight: '400',
+  },
+  backButton: {
+    marginTop: 100,
+    marginBottom: 40,
+    marginRight: 280,
+    padding: 10,
+  },
+  instructionText: {
+    color: '#000',
+    fontSize: 24,
+    fontWeight: '800',
+    marginRight: 18,
+    marginLeft: 18,
+  },
+  passwordText: {
+    marginTop: 115,
+    marginRight: 230,
+    fontSize: 13,
+    opacity: 0.35,
+  },
+  input: {
+    marginTop: 5,
+    height: 40,
+    borderBottomWidth: 2,
+    padding: 10,
+    width: 310,
+    borderRadius: 5,
+    borderColor: '#920E1A',
+  },
+  nextText: {
+    marginRight: 200,
+    fontSize: 14,
+    opacity: 1,
+    fontWeight: '400',
+  },
+  nextButton: {
```

Comment:
Same comment as above but this should be black if there is input, otherwise grey like you have right now

Reply:
Resolving because we can ignore for now